testsuite/scripts/src/test/java/org/wildfly/scripts/test/StandaloneScriptTestCase.java

```diff
@@ -100,6 +119,46 @@ void testScript(final ScriptProcess script) throws InterruptedException, Timeout
             Assert.assertFalse("Did not expect to find -Djava.security.manager=allow in the JVM parameters.", stdout.contains("-Djava.security.manager=allow"));
         }
 
+        // test WFCORE-5917
+        if (!script.getContainerHome().toString().contains(" ")) {
+            ProcessHandle handle = script.toHandle();
+            List<ProcessHandle> children = handle.children().collect(Collectors.toList());
+
+            String launchCommand = "";
+            if (TestSuiteEnvironment.isWindows()) {
+                if (isPowerShellSupported()) {
```

Comment:
Use `Shell.POWERSHELL.isSupported()` instead, it is not necessary to repeat the same logic in this class by creating `isPowerShellSupported` method